doc/modules/model_evaluation.rst

```diff
@@ -1614,12 +1614,12 @@ better suited.
 
   .. [Martin1997] A. Martin, G. Doddington, T. Kamm, M. Ordowski, and M. Przybocki,
      `The DET Curve in Assessment of Detection Task Performance
-     <http://www.dtic.mil/docs/citations/ADA530509>`_,
+     <https://www.researchgate.net/publication/303637188_The_det_curve_in_assessment_of_detection_task_performance>`_,
      NIST 1997.
 
   .. [Navratil2007] J. Navractil and D. Klusacek,
      "`On Linear DETs,
-     <http://www.research.ibm.com/CBG/papers/icassp07_navratil.pdf>`_"
+     <https://www.researchgate.net/publication/4249280_On_Linear_DETs>`_"
```

Comment:
I can't find the a publicly available resource, but we can link directly to the publisher:

```suggestion
     <https://ieeexplore.ieee.org/document/4218079>`_"
```